### PR TITLE
Updated useErrorToast with AntD notification; closes #2396.

### LINKED ISSUE #2396: Replace @blueprintjs Toaster with AntD Notifications
- see https://ant.design/components/notification/

- would be nice if existing toaster hooks API could be maintained to reduce refactoring required in other functions and components, but feel free to look at usage within existing locations like Apollo, and if it will be easier and faster to just strip it all out and start from scratch, then just do that and/or feel free to change accordingly if you see room for any improvement (ie: better functionality, etc)

- convert existing toasts as follows:
    - useSuccessToast() => notification.success()
    - useErrorToast() => notification.error()
    - useInfoToast() => notification.info()
    - useLoadingToast() => notification.open() - with loading-3-quarters icon
    - useDismissToast() => notification.close() - OR... look at the "Update Message Content" section and consider using that for changing notices in some scenarios, such as when entity loading is completed or a CRUD operation finishes, instead of closing the previous notice and opening a second one

- also add new
    - useWarningToast() => notification.warning()

- keep notifications in the bottom right corner by default, but allow that to be controlled via API

- you can likely keep the ToasterHash function for generating message IDs but feel free to change if you have an easier/better idea

### PR DESCRIPTION
**Pull request by nicolad**
## Problem this Pull Request solves
- Updates `useErrorToast`;
- Updates `useInfoToast`;
- Adds `useWarningToast`;
- More info: https://github.com/eventespresso/event-espresso-core/issues/2396

### CODE REVIEW

File: assets/src/application/services/toaster/useErrorToast.tsx [hunk 1]
```diff
@@ -1,39 +1,40 @@
...
 const useErrorToast: ErrorToast = (toaster, hash) =>
 	useCallback<ErrorToastCallback>(
-		(message, timeout = 5000, action = {}, onDismiss = null) => {
-			let icon: ErrorIcon = 'warning-sign';
+		({ message = '', placement = 'bottomRight', ...rest }) => {
+			let icon = (
+				<Icon
+					type='close-circle'
+					theme='twoTone'
+					twoToneColor='#eb2f96' // temp
```
Comment (tn3rb) [inline assets/src/application/services/toaster/useErrorToast.tsx]:
```suggestion
					twoToneColor='var(--ee-color-accent)'
```

File: package.json [hunk 2]
```diff
@@ -74,7 +74,7 @@
 		"@wordpress/i18n": "^3.9.0",
 		"@wordpress/is-shallow-equal": "^1.8.0",
 		"accounting-js": "^1.1.1",
-		"antd": "^3.26.9",
+		"antd": "^4.0.0-rc.5",
```
Comment (nicolad) [inline package.json]: This is the latest version:
image

File: assets/src/application/services/toaster/useErrorToast.tsx [hunk 3]
```diff
@@ -1,39 +1,36 @@
 import React, { useCallback } from 'react';
 import { ApolloError } from 'apollo-client';
-import { Intent } from '@blueprintjs/core';
+import { notification } from 'antd';
+import Icon, { CloseCircleTwoTone, GlobalOutlined } from '@ant-design/icons';
```
Comment (nicolad) [inline assets/src/application/services/toaster/useErrorToast.tsx]: Based on demo: https://github.com/ant-design/ant-design/blob/4.0.0-rc.5/components/icon/demo/two-tone.md

File: assets/src/application/services/toaster/useErrorToast.tsx [hunk 4]
```diff
@@ -1,39 +1,36 @@
...
+		({ message = '', placement = 'bottomRight', ...rest }) => {
+			let icon = <CloseCircleTwoTone twoToneColor='var(--ee-color-accent)' />;
+
 			let errorMessage: ToasterMsg;
+
 			if (message instanceof ApolloError) {
 				errorMessage = message.message;
 				if (message.graphQLErrors) {
-					icon = <IconGraphQL />;
+					icon = <Icon component={IconGraphQL} />;
```
Comment (nicolad) [inline assets/src/application/services/toaster/useErrorToast.tsx]: Based on: https://github.com/ant-design/ant-design/blob/4.0.0-rc.5/components/icon/demo/custom.md

Comment (manzoorwanijk) [inline assets/src/application/services/toaster/useErrorToast.tsx]: Cool 👍

**Pull request merged**